### ISSUE TITLE
A way for text in the extension to save when browser/extension is closed.

### ISSUE THREAD

**Issue by LukeG294** [LukeG294/TextCheck#2]
**Is your feature request related to a problem? Please describe.**
All text is cleared when extension is clicked out of.

**Describe the solution you'd like**
A way for the text entered in the text box to save. Possibly a save button?

**Describe alternatives you've considered**
N/A

**Additional context**
You can do it however you'd like, all the files you need to edit are on the "code" page.

**Comment by preeti13456** [comment]
in chrome?

**Comment by LukeG294** [comment]
Yeah, it's a Chrome extension. Not sure if it's possible but it's a little frustrating when all the text disappears.

**Comment by LukeG294** [comment]
Here's the current HTML code: https://github.com/LukeG294/TextCheck/blob/main/hello.html

**Comment by preeti13456** [comment]
ok I can try

**Comment by preeti13456** [comment]
Can we use manifest.json and edit permission will that work?

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by LukeG294** [comment]
Yes, that should work. You should be able to make a fork of the manifest file and then make a pull request that I can manually accept and update the extension.

**Comment by LukeG294** [comment]
https://github.com/LukeG294/TextCheck/blob/main/manifest.json

**Comment by preeti13456** [comment]
Yes surely will do

Get Outlook for Android<https://aka.ms/ghei36>

________________________________
